Realm/RLMObservation.hpp

```diff
@@ -178,3 +181,18 @@ class RLMObservationTracker {
 std::vector<realm::BindingContext::ObserverState> RLMGetObservedRows(RLMSchemaInfo const& schema);
 void RLMWillChange(std::vector<realm::BindingContext::ObserverState> const& observed, std::vector<void *> const& invalidated);
 void RLMDidChange(std::vector<realm::BindingContext::ObserverState> const& observed, std::vector<void *> const& invalidated);
+
+// ???: Will not using RLM conflict with other libraries?
+realm::KeyPath KeyPathFromString(RLMRealm *realm,
```

Comment:
Will this need RLM- prefix?

Reply:
Everything except for static functions and functions defined inside an anonymous namespace should have the RLM prefix.